In [1]:
from ultralytics import YOLO
import time

In [2]:
ZenoC = YOLO("yolo11n-obb.pt")
start =time.time()
test = ZenoC("Akshay Kumar_0.jpg",imgsz=320, workers =8)
end = time.time()
test[0].show()
print(f"Total Time:{end-start:.3f} seconds")


image 1/1 F:\RNN based Object detection and Anomaly Classification surveillance System\Akshay Kumar_0.jpg: 320x224 (no detections), 72.8ms
Speed: 1.4ms preprocess, 72.8ms inference, 0.9ms postprocess per image at shape (1, 3, 320, 224)
Total Time:0.225 seconds


In [3]:
ZenoC.export(format="openvino",dynamic=True)

Ultralytics 8.3.196  Python-3.12.7 torch-2.8.0+cpu CPU (Intel Core i5-8250U 1.60GHz)

PyTorch: starting from 'yolo11n-obb.pt' with input shape (1, 3, 1024, 1024) BCHW and output shape(s) (1, 20, 21504) (5.5 MB)
requirements: Ultralytics requirement ['openvino>=2024.0.0'] not found, attempting AutoUpdate...
   ---------------------------------------- 0.0/40.6 MB ? eta -:--:--
   ---------------------------------------- 0.3/40.6 MB ? eta -:--:--
   ---------------------------------------- 0.3/40.6 MB ? eta -:--:--
    --------------------------------------- 0.5/40.6 MB 1.0 MB/s eta 0:00:39
    --------------------------------------- 0.8/40.6 MB 1.0 MB/s eta 0:00:38
   - -------------------------------------- 1.0/40.6 MB 1.1 MB/s eta 0:00:35
   - -------------------------------------- 1.3/40.6 MB 1.2 MB/s eta 0:00:33
   - -------------------------------------- 1.6/40.6 MB 1.2 MB/s eta 0:00:34
   - -------------------------------------- 1.8/40.6 MB 1.2 MB/s eta 0:00:32
   -- --------------

'yolo11n-obb_openvino_model'

image 1/1 F:\RNN based Object detection and Anomaly Classification surveillance System\Akshay Kumar_0.jpg: 320x224 (no detections), 34.2ms
Speed: 0.9ms preprocess, 34.2ms inference, 0.6ms postprocess per image at shape (1, 3, 320, 224)
Total Time:0.120 second